Razorpay.php

```diff
@@ -1,10 +1,13 @@
 <?php
 
-// Include Requests
-require_once __DIR__.'/libs/Requests-1.6.1/library/Requests.php';
+// Include Requests only if not already defined
```

Comment:
The only concern I have with this is that Requests could have been defined as any class, not specifically the class we are looking for. So far we've dealt with conflicts in only one place: WordPress, where commenting out this line works as a workaround, but if someone happens to have a Requests class defined that is not what we are expecting, this will throw up weirder errors than class defined again.

Might work (since we haven't seen a conflict with another class called Requests yet) out well though.